tests/train_test.py

```diff
@@ -1,7 +1,13 @@
+import pytest
 from click.testing import CliRunner
 from larq_zoo import train
+import tensorflow as tf
 
 
+@pytest.mark.skipif(
+    int(tf.__version__[0]) == 2,
+    reason="This currently fails on TF 2 due to https://github.com/larq/larq/pull/195.",
+)
```

Comment:
Should we test for tf version when making the model for which tf2 fails and raise a comprehensible error preemptively?

Reply:
I don't think that's necessary for now, since TF 2 isn't stable yet and this might be a bug in TensorFlow itself. Only the training code fails, so loading the model via the public API works correctly in this version.

We can add a proper error message once we have time to dig into https://github.com/larq/larq/pull/195